src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -40,14 +41,10 @@
     <Compile Include="System\Runtime\Serialization\SerializationException.cs" />
     <Compile Include="System\Runtime\Serialization\StreamingContext.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'==''">
+  <ItemGroup Condition="'$(TargetGroup)'=='' Or '$(TargetGroup)'=='uap101aot'">
     <Compile Include="System\Runtime\Serialization\InvalidDataContractException.Serialization.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'==''">
-    <!-- ToDo: Remove this dependency once System.Runtime is published. -->
-    <ProjectReference Include="..\..\System.Runtime\pkg\System.Runtime.pkgproj">
-      <Name>System.Runtime</Name>
-    </ProjectReference>
+  <ItemGroup Condition="'$(TargetGroup)'=='' Or '$(TargetGroup)'=='uap101aot'">
```

Comment:
More or less. What I mean is that if you have these types in the reference assembly, in theory GenFacades should be adding the type forwards for the implementation automatically. I'm just trying to understand why do we force the c# compiler to add the typeforwards instead of having genfacades do it.